qwandaq/src/main/java/life/genny/qwandaq/validation/Validation.java

```diff
@@ -296,4 +297,18 @@ public CoreEntitySerializable toSerializableCoreEntity() {
 		validation.setSelectionGroup(validationListConverter.convertToString(getSelectionBaseEntityGroupList()));
 		return validation;
 	}
+
+	@Override
+	public int hashCode() {
+		HashCodeBuilder hcb = new HashCodeBuilder();
+		hcb.append(getRealm());
+		hcb.append(getCode());
+		return hcb.toHashCode();
```

Comment:
This method is shadowing the CodedEntity@hashCode method which has the exact same implementation, so this method can be removed all together

Reply:
Fixed